File: Java/Movie.java
public class Movie {
  private String title, director, lang;
  private int minutes, year, price;

  public Movie() {}

  public Movie(String t, String d, String l, int m, int y, int p) {
    setTitle(t);
    setDirector(d);
    setLang(l);
    setMinutes(m);
    setYear(y);
    setPrice(p);
  }

  public int setTitle(String t) {
    if(t == null || t.isEmpty()) {
      System.out.println("Judul tidak bisa kosong!");
      return -1;
    }

    title = t;
    return 0;
  }
  public String getTitle() {
    return title;
  }

  public void setDirector(String d) {
    if(d == null || d.isEmpty()) {
      director = "-";
    } else {
      director = d;
    }
  }
  public String getDirector() {
    return director;
  }

  public void setLang(String l) {
    if(l == null || l.isEmpty()) {
      lang = "-";
    } else {
      lang = l;
    }
  }
  public String getLang() {
      return lang;
  }

  public int setMinutes(int m) {
    if(m < 0) {
      System.out.println("Durasi harus bilangan bulat positif!");
      return -1;
    }

    minutes = m;
    return 0;
  }
  public int getMinutes() {
    return minutes;
  }

  public int setYear(int y) {
    if(y < 0) {
      System.out.println("Tahun rilis harus bilangan bulat positif!");
      return -1;
    }

    year = y;
    return 0;
  }
  public int getYear() {
    return year;
  }

  public int setPrice(int p) {
    if(p < 0) {
      System.out.println("Harga harus bilangan bulat positif!");
      return -1;
    }

    price = p;
    return 0;
  }
  public int getPrice() {
    return price;
  }
}

File: PHP/Movie.php
<?php

class Movie {
  private string $title, $director, $lang, $img_url;
  private int $id, $minutes, $year, $price;

  public function __construct(int $id, string $t, string $d, string $l, int $m, int $y, int $p, string $i) {
    $this->setId($id);
    $this->setTitle($t);
    $this->setDirector($d);
    $this->setLang($l);
    $this->setMinutes($m);
    $this->setYear($y);
    $this->setPrice($p);
    $this->setImgUrl($i);
  }

  public function setId(int $id): void {
    $this->id = $id;
  }
  public function getId(): int {
    return $this->id;
  }

  public function setTitle(string $t): void {
    $this->title = $t;
  }
  public function getTitle(): string {
    return $this->title;
  }

  public function setDirector(string $d): void {
    if ($d === "") {
      $this->director = "-";
    } else {
      $this->director = $d;
    }
  }
  public function getDirector(): string {
    return $this->director;
  }

  public function setLang(string $l): void {
    if ($l === "") {
      $this->lang = "-";
    } else {
      $this->lang = $l;
    }
  }
  public function getLang(): string {
    return $this->lang;
  }

  public function setMinutes(int $m): void {
    $this->minutes = $m;
  }
  public function getMinutes(): int {
    return $this->minutes;
  }

  public function setYear(int $y): void {
    $this->year = $y;
  }
  public function getYear(): int {
    return $this->year;
  }

  public function setPrice(int $p): void {
    $this->price = $p;
  }
  public function getPrice(): int {
    return $this->price;
  }

  public function setImgUrl(string $i): void {
    if ($i === "") {
      $this->img_url = "-";
    } else {
      $this->img_url = $i;
    }
  }
  public function getImgUrl(): string {
    return $this->img_url;
  }
}